VSCode/armsnippets.json

```diff
@@ -15,6 +15,30 @@
     ],
     "description": "Skeleton ARM Template"
 },
+"Azure Resource Manager (ARM) Parameters Template": {
+    "prefix": "armp!",
+    "body": [
+        "{",
+        "    \"$$schema\": \"https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#\",",
+        "    \"contentVersion\": \"1.0.0.0\",",
+        "    \"parameters\": {}",
+        "}"
+    ],
+    "description": "Skeleton ARM Parameters Template"
+},
+"Azure Resource Manager (ARM) Metadata Template": {
```

Comment:
This metadata isn't specific to ARM templates, so I'd rather not add it to the ARM snippets. Maybe it belongs in a more general snippet collection?

Reply:
Will remove that one then